### ISSUE TITLE
Allow override for other-config:perfmon on control domain during install

### ISSUE THREAD

**Issue by NeilHanlon** [xcp-ng/xcp#462]
`firstboot.d/50-prepare-control-domain-params` in the xenserver-firstboot package sets some (probably sane) defaults for the perfmon config on dom0.

In a large infrastructure, this perfmon config just causes noise and, ultimately, we think slowdowns both in XO, as well as on the hosts (probably, due to huge XAPI DB sizes and IO overhead writing and dealing with messages we do not care about).

We would like to be able to incorporate overriding this config (`xe vm-param-set uuid=$uuid other-config:perfmon='<config></config>'`) -- basically just customizing it.. into our autoinstalls.

Basically, we'd like to be able to have an answerfile option that we deliver to the host in our kickstart process which provides the customization.

@ me if more information is needed!

Thanks,
Neil

**Comment by stormi** [comment]
Enhancement suggestion duly noted.

In the meantime, you can probably already do it specifying a post-install script directly in the answerfile.

Example :
```
<script stage="filesystem-populated" type="url">
http://pxe/configs/presets/scripts/filesystem-populated.py
</script>
```

See https://github.com/xcp-ng/host-installer/blob/master/doc/answerfile.txt#L104

**Comment by NeilHanlon** [comment]
Perfect! Forgot about that option.

It _might_ make more sense for this to be just an xo plug-in.. Or both. If
the data were exposed "better" via xo api, I think that may be sufficient.
(currently no api I can find to vm-param-set on dom0, at least not these
attributes).

An xo plug-in is interesting as it'd allow xo to be a nms of sorts setting
alarms on all vms and such. Just spit balling.
